### PR TITLE
Sslabusech feed

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

Waiting for an image + default values from Ryan

### CODE REVIEW

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        'https://sslbl.abuse.ch/blacklist/sslipblacklist.csv': {
+            'fieldnames': ['date_created', 'value', 'port'],
+            'indicator_type': FeedIndicatorType.IP
+        },
+        'https://sslbl.abuse.ch/blacklist/sslipblacklist_aggressive.csv': {
+            'fieldnames': ['date_created', 'value', 'port'],
+            'indicator_type': FeedIndicatorType.IP
+        }
+    }
+
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.py]: We'll need to add a mapping to this here, correct?

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  - https://sslbl.abuse.ch/blacklist/sslipblacklist_aggressive.csv
+  required: true
+  type: 16
+- display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+  defaultvalue: true
+- defaultvalue: '360'
+  display: Fetch Interval
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  display: Feed Fetch Interval
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  name: feedFetchInterval
+  required: false
+  type: 19
+- defaultvalue: B - Usually reliable
+  display: Reliability
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  display: Source Reliability
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  name: feedReliability
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: false
+  type: 15
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  type: 15
  additionalinfo: Reliability of the source providing the intelligence data
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+- additionalinfo: Indicators from this integration instance will be marked with this
+    reputation.
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
    reputation
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  defaultvalue: Bad
+  display: Indicator reputation
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  display: Indicator Reputation
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+  name: feedReputation
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  required: false
+  type: 18
+- defaultvalue: indicatorType
+  display: expirationPolicy
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  display: ""
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+  name: feedExpirationPolicy
+  required: false
+  type: 17
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  type: 17
  options:
  - never
  - interval
  - indicatorType
  - suddenDeath
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+- display: expirationInterval
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
- display: ""
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+  name: feedExpirationInterval
+  required: false
+  type: 1
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  type: 1
  defaultvalue: "20160"
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+  defaultvalue: '20'
+  display: Request Timeout
+  name: polling_timeout
+  required: true
+  type: 0
+- additionalinfo: When selected, the exclusion list will be ignored for this feed.
+  display: Bypass exclusion list
+  name: feedBypassExclusionList
+  required: false
+  type: 8
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
  type: 8
  defaultvalue: ""
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+  name: feedExpirationInterval
+  required: false
+  type: 1
+- additionalinfo: Time in seconds before http requests timeout.
+  defaultvalue: '20'
+  display: Request Timeout
+  name: polling_timeout
+  required: true
+  type: 0
+- additionalinfo: When selected, the exclusion list will be ignored for this feed.
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
- additionalinfo: When selected, the exclusion list is ignored for indicators from
    this feed. This means that if an indicator from this feed is on the exclusion
    list, the indicator might still be added to the system.
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Use the SSLBL feed integration to fetch indicators from the feed.
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]:
```suggestion
description: Use the SSLBL feed integration to fetch indicators from the SSL Abuse Blacklist feeds.
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 14]
```diff
@@ -0,0 +1,114 @@
...
+    description: Gets the feed indicators.
+    execution: false
+    name: sslbl-get-indicators
+    outputs:
+    - contextPath: BambenekConsultingFeed.Indicator.Value
+      description: The indicator value.
+      type: String
+    - contextPath: BambenekConsultingFeed.Indicator.Type
+      description: The indicator type.
+      type: String
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]: Need to be removed... walla walla whoops.

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml [hunk 15]
```diff
@@ -0,0 +1,114 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: SSLBL Feed
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.yml]: Let's change the `id`, `name`, and `display` fields - `SSLBL` is not a good name IMO. Maybe `SSL Blacklist` or `SSL Abuse Blacklist`... I'm not sure. Whatever you decide, the filenames should be updated accordingly.

File: Packs/Feedsslabusech/TestPlaybooks/playbook-sslbl_test.yml [hunk 16]
```diff
@@ -0,0 +1,186 @@
+elasticcommonfields: {}
+id: sslbl test
+version: 1
```
Comment (avidan-H) [inline Packs/Feedsslabusech/TestPlaybooks/playbook-sslbl_test.yml]:
```suggestion
version: -1
```

File: Packs/Feedsslabusech/TestPlaybooks/playbook-sslbl_test.yml [hunk 17]
```diff
@@ -0,0 +1,186 @@
...
+      "dimensions": {
+        "height": 735,
+        "width": 380,
+        "x": 50,
+        "y": 50
+      }
+    }
+  }
+inputs: []
+outputs: []
```
Comment (avidan-H) [inline Packs/Feedsslabusech/TestPlaybooks/playbook-sslbl_test.yml]:
```suggestion
outputs: []
fromversion: 5.5.0
```

File: Tests/conf.json [hunk 18]
```diff
@@ -195,6 +195,10 @@
             "integrations": "Minerva Labs Anti-Evasion Platform",
             "playbookID": "Minerva Test playbook"
         },
+        {
+            "integrations": "SSLBL Feed",
+            "playbookID": "sslbl test"
```
Comment (avidan-H) [inline Tests/conf.json]:
```suggestion
            "playbookID": "sslbl test",
            "fromversion": "5.5.0"
```

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.py (hunk 1 above)
Comment (ronykoz) [reply Packs/Feedsslabusech/Integrations/Feedsslabusech/Feedsslabusech.py]: right

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md [hunk 19]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+Use the AWS feed integration to fetch indicators from the feed.
+
+
+## Configure AWS Feed on Demisto
+---
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for SpamhausFeed.
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md]: This says "Sphamhaus"

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md [hunk 20]
```diff
@@ -0,0 +1,31 @@
+## Overview
+---
+
+Use the AWS feed integration to fetch indicators from the feed.
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md]: This says AWS

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md [hunk 21]
```diff
@@ -0,0 +1,31 @@
...
+
+## Configure AWS Feed on Demisto
+---
+
+1. Navigate to __Settings__ > __Integrations__ > __Servers & Services__.
+2. Search for SpamhausFeed.
+3. Click __Add instance__ to create and configure a new integration instance.
+    * __Name__: a textual name for the integration instance.
+    * __Sub-Feeds__: Sub-Feeds of AWS to fetch indicators from: 
+        * AMAZON - All AMAZON ranges.
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md]: Sub feeds are incorrect

File: Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+        * EC2 - EC2 ranges.
+        * ROUTE53 - ROUTE53 ranges. 
+        * ROUTE53_HEALTHCHECKS - ROUTE53_HEALTHCHECKS ranges.
+        * CLOUDFRONT - CLOUDFRONT ranges.
+        * S3 - S3 ranges.
+    * __Fetch indicators__: boolean flag. If set to true will fetch indicators.
+    * __Fetch Interval__: Interval of the fetches.
+    * __Reliability__: Reliability of the feed.
```
Comment (avidan-H) [inline Packs/Feedsslabusech/Integrations/Feedsslabusech/README.md]:
```suggestion
    * __Source Reliability__: Reliability of the feed.  
```

**Pull request merged**